src/controllers/request-controller.ts

```diff
@@ -117,7 +117,7 @@ export default class RequestController implements Contextual {
   }
 
   @Post()
-  private async create(req: ExpReq, res: ExpRes): Promise<ExpRes<any>> {
+  private async createRequest(req: ExpReq, res: ExpRes): Promise<ExpRes<any>> {
```

Comment:
why renaming it to `createRequest`? I guess it's obvious since it's in the `RequestController`.